components/script/webdriver_handlers.rs

```diff
@@ -185,6 +180,45 @@ pub fn handle_get_title(page: &Rc<Page>, _pipeline: PipelineId, reply: IpcSender
     reply.send(String::from(page.document().Title())).unwrap();
 }
 
+// https://w3c.github.io/webdriver/webdriver-spec.html#get-element-rect
+pub fn handle_get_rect(page: &Rc<Page>,
+                       pipeline: PipelineId,
+                       element_id: String,
+                       reply: IpcSender<Result<Rect<f64>, ()>>) {
+    reply.send(match find_node_by_unique_id(&*page, pipeline, element_id) {
+        Some(elem) => {
+            // https://w3c.github.io/webdriver/webdriver-spec.html#dfn-calculate-the-absolute-position
+            match elem.downcast::<HTMLElement>() {
+                Some(html_elem) => {
+                    // Step 1
```

Comment:
This will definitely confuse people as to which spec this step is following. I think this should only link to the absolute position spec.